pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -555,6 +557,40 @@ func getContainer(el *v1alpha1.EventListener, c Config) corev1.Container {
 	}
 }
 
+func getPort(el *v1alpha1.EventListener, c Config) int {
+	var (
+		elCert, elKey string
+	)
+
+	certEnv := map[string]*corev1.EnvVarSource{}
+	if el.Spec.Resources.KubernetesResource != nil {
+		if len(el.Spec.Resources.KubernetesResource.Template.Spec.Containers) != 0 {
+			for i := range el.Spec.Resources.KubernetesResource.Template.Spec.Containers[0].Env {
+				certEnv[el.Spec.Resources.KubernetesResource.Template.Spec.Containers[0].Env[i].Name] =
+					el.Spec.Resources.KubernetesResource.Template.Spec.Containers[0].Env[i].ValueFrom
+			}
+		}
+	}
+
+	if v, ok := certEnv["TLS_CERT"]; ok {
+		elCert = v.SecretKeyRef.Key
+	} else {
+		elCert = ""
+	}
+	if v, ok := certEnv["TLS_KEY"]; ok {
+		elKey = v.SecretKeyRef.Key
+	} else {
+		elKey = ""
+	}
+
+	if elCert != "" && elKey != "" {
+		// We always set port 8443 if TLS is enabled - regardless of the configured port.
+		return 8443
```

Comment:
This was discussed on slack to be the intended behaviour. Please let me know if you think this should be different.

Reply:
hmmm..do we know why we can't/shouldn't change the port if TLS is enabled? @savitaashture

Reply:
No specific reason but I see TLS connections usually enabled on `443` or `8443`
and for `443` port getting error related to permission bind
```
{"level":"fatal","ts":"2021-01-04T17:05:15.398Z","logger":"eventlistener","caller":"eventlistenersink/main.go:145","msg":"failed to start eventlistener sink: listen tcp :443: bind: permission denied","knative.dev/controller":"eventlistener","stacktrace":"main.main\n\tgithub.com/tektoncd/triggers/cmd/eventlistenersink/main.go:145\nruntime.main\n\truntime/proc.go:204"}
```
So just used `8443`

Reply:
I would then prefer to create a separate default for the TLS connection?
So users have the ability to overwrite it.

Reply:
Sure i will send a PR to have default port for TLS

then we can also merge this https://github.com/tektoncd/triggers/pull/880 and i will add different PortName for TLS

Reply:
@MarcelMue do you want to handle this as part of this PR itself ?
Or you want me to send separate PR

Reply:
I will take care in this PR. Change coming up in a few minutes :)

Reply:
@savitaashture Can you check? Changes are in now.

Reply:
ok, this looks fine for now. I'll add an item to discuss the separate TLS port behavior in the next working group.